fhir-bucket/pom.xml

```diff
@@ -54,6 +54,10 @@
             <groupId>org.apache.httpcomponents</groupId>
             <artifactId>httpcore</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.apache.derby</groupId>
+            <artifactId>derby</artifactId>
```

Comment:
Could the scope be test here?

Reply:
it was previously a compile scope (but inhereted from other modules).  I think we build a cli (all-in-one) jar out of it, so we'd need to make sure that works before changing it...I'd opt to save this for some future work item.